libs/tools/barista/src/transform.ts

```diff
@@ -202,6 +202,27 @@ export const copyHeadlineTransformer: BaPageTransformer = async source => {
   return transformed;
 };
 
+export const internalLinksToRouterLinksTransformer: BaPageTransformer = async source => {
+  const transformed = { ...source };
+  if (source.content && source.content.length) {
+    transformed.content = runWithCheerio(source.content, $ => {
+      const links = $('a');
+      if (links.length) {
+        links.each((_, link) => {
+          const linkValue = $(link).attr('href');
+          if (linkValue && !linkValue.includes('http')) {
```

Comment:
I would update the check here. An internal link could always include `http` somewhere. Either change it to `.startsWith('http')` or check if the link starts with `/` followed by a letter, then it should be an internal link. Or maybe check if the linkValue contains `://`, then it's also an absolute URL and probably not linking to another internal page.

Reply:
Exactly what I needed to know. Thank you